app/assets/src/components/views/SampleView/SampleViewControls.jsx

```diff
@@ -34,7 +35,13 @@ class SampleViewControls extends React.Component {
 
   deleteSample = async () => {
     const { sample, project } = this.props;
+    let nanobar = new Nanobar({
+      id: "prog-bar",
+      class: "prog-bar",
+    });
+    nanobar.go(30);
     await deleteSample(sample.id);
```

Comment:
a further improvement would be to console.log any errors or display them but this is fine

Reply:
thx